app/models/Session/QuestionTypes.js

```diff
@@ -162,6 +176,18 @@ const notAddableQuestionTypes = [
   tlfbQuestion.id,
 ];
 
-export const AddableQuestionTypes = QuestionTypes.filter(
-  ({ id }) => !notAddableQuestionTypes.includes(id),
+const addableToSmsSessionQuestionTypes = [
+  smsQuestion.id,
+  smsInformationQuestion.id,
+];
+
+export const ClassicSessionQuestionTypes = QuestionTypes.filter(
+  ({ id }) =>
+    !notAddableQuestionTypes
+      .concat(addableToSmsSessionQuestionTypes)
+      .includes(id),
+);
+
+export const SmsSessionQuestionTypes = QuestionTypes.filter(({ id }) =>
+  addableToSmsSessionQuestionTypes.includes(id),
```

Comment:
Please reconsider naming of these arrays because they contain only question types that can be **added** in specific types of sessions but currently they are named as if they would contain **all** question types that can be in that sessions